coral-spark/src/main/java/com/linkedin/coral/spark/CoralSpark.java

```diff
@@ -70,6 +70,22 @@ public static CoralSpark create(RelNode irRelNode) {
     return new CoralSpark(baseTables, sparkUDFInfos, sparkSQL);
   }
 
+  /**
+   * Users use this function to get CoralSqlNode from CoralRelNode
+   * This should be used when user need to get CoralSqlNode from CoralRelNode by applying
+   * spark specific transformations on CoralRelNode
+   * with Coral-schema output schema
+   *
+   * @return [[SqlNode]]
+   */
```

Comment:
Java doc needs to be more elaborate, and reflect the relationship with existing methods. @aastha25 could help with that.